### PR TITLE
New branch: made changes to _Western.less and incorporated under special instead of animated category

### PR DESCRIPTION
**Pull request by laurachristensen1214**
Issue:

### CODE REVIEW

**Review by shahednasser — CHANGES_REQUESTED** [review]
Thank you for your work @laurachristensen1214 but I can't see how the button looks like because you still need to run `npm run compile` then commit the changes.

Also, Please remove the file changes in `.vs/VSWorkspaceState.json` and `.vs/slnx.sqlite` as they are not necessary

**Comment by laurachristensen1214** [comment]
Hi @shahednasser I have resynced and compiled and made some changes to mine after looking at others'.

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -13,7 +13,7 @@ Please note we have a code of conduct, please follow it in all your interactions
 # Contents
 
 - [Instructions](#instructions)
-  * [Button Ideas](#button-ideas)
+  * [Button Ideas](#button-ideas)ru
```
Comment (shahednasser) [inline CONTRIBUTING.md:16]: remove this change

File: src/components/special/_western.less [hunk 2]
```diff
@@ -0,0 +1,65 @@
...
+  border: 1px dashed rgba(@creamy, 60%);
+
+  &:hover {
+    transition: all 150ms linear;
+    text-decoration: none;
+    box-shadow: 2px 2px 4px rgba(#fff, 0.2);
+    border: 1px dashed darken(@gold, 4%);
+  }
+
+  &-denim {
```
Comment (shahednasser) [inline src/components/special/_western.less:13]: Change this to `&.denim`

File: src/components/special/_western.less [hunk 3]
```diff
@@ -0,0 +1,65 @@
...
+    color: lighten(@denim, 35%);
+
+    &:hover {
+      background: darken(@denim, 1.5%);
+      color: lighten(@denim, 18%);
+      text-shadow: -1px -1px 0 darken(@denim, 9.5%);
+    }
+  }
+
+  &-choco {
```
Comment (shahednasser) [inline src/components/special/_western.less:26]: Change this to &.choco

File: src/components/special/_western.less [hunk 4]
```diff
@@ -0,0 +1,65 @@
...
+    color: lighten(@choco, 55%);
+
+    &:hover {
+      background: darken(@choco, 1.5%);
+      color: lighten(@choco, 18%);
+      text-shadow: -1px -1px 0 darken(@choco, 9.5%);
+    }
+  }
+
+  &-charcoal {
```
Comment (shahednasser) [inline src/components/special/_western.less:39]: Change this to &.charcoal

File: src/components/special/_western.less [hunk 5]
```diff
@@ -0,0 +1,65 @@
...
+    color: lighten(@charcoal, 55%);
+
+    &:hover {
+      background: darken(@charcoal, 1.5%);
+      color: lighten(@charcoal, 18%);
+      text-shadow: -1px -1px 0 darken(@charcoal, 9.5%);
+    }
+  }
+
+  &-sand {
```
Comment (shahednasser) [inline src/components/special/_western.less:53]: Change this to &.sand

File: src/_variables.less [hunk 6]
```diff
@@ -125,6 +125,15 @@
 @toggleSwitchOnPosition: translateX(calc(@toggleWidth * 0.4));
 /* END VARIABLES IN special/_toggle.less */
 
+/* BEGIN VARIABLES IN special/_western.less*/
+@choco: #59452b;
```
Comment (shahednasser) [inline src/_variables.less:129]: Please rename all your variables to start with western (for example `@westernChoco`) that makes it easier to maintain later on.

**Review by shahednasser — CHANGES_REQUESTED** [review]
Thank you for your work @laurachristensen1214 but I have some comments below. Can you please also remove all the changes in `.vs/slnx.sqlite` and `.vs/VSWorkspaceState.json` and `package-lock.json` and `package.json`? It makes the PR messy.

I realize that hacktoberfest is coming to an end soon and so I'll be adding `hacktoberfest-accepted` to your PR to make sure it's accepted even if it's not merged in time.

**Comment by laurachristensen1214** [comment]
I have made changes and checked into new-branch. Sorry my repo seems to be such a mess! I'm trying to straighten it out a bit so it's not so confusing. And thank you for adding the accepted tag. 👍

**Review by shahednasser — CHANGES_REQUESTED** [review]
Ok now I can see the buttons, can you please make the borders more inside the button? What I mean by that is maybe you can make use of the `:before` or `:after` instead of applying the border directly to the button as it will make it look better.

Also please remove the changes in the files I've mentioned before

**Comment by shahednasser** [comment]
Hi @laurachristensen1214 are you still working on this?

**Pull request closed without merging**